packages/core/src/middleware/defaults/feeEstimator.ts

```diff
@@ -18,7 +18,7 @@ export const defaultFeeEstimator: <C extends MiddlewareClient>(
     // for more information about maxFeePerGas and maxPriorityFeePerGas
 
     const feeData = await client.estimateFeesPerGas();
-    if (!feeData.maxFeePerGas || !feeData.maxPriorityFeePerGas) {
+    if (!feeData.maxFeePerGas || feeData.maxPriorityFeePerGas == null) {
```

Comment:
we should be consistent with both fields if we're gonna make this change
```suggestion
    if (feeData.maxFeePerGas == null || feeData.maxPriorityFeePerGas == null) {
```